content/tutorials/manage-the-rippled-server/troubleshooting/server-doesnt-sync.md

```diff
@@ -0,0 +1,110 @@
+
+The `validator_list` block in the [server_info method][] response shows the status of your validator list including its expiration date. If you have a list but it's expired, it's possible that your server had connectivity to the validator list site before but hasn't been able to connect lately, so your current list expired while your server was unable to download a more updated list.
+
+You can also use the [validator_list_sites method][] to get more detailed information. If the `last_refresh_status` and `last_refresh_time` fields are missing from the validator site objects in the response, that probably indicates that your server is having trouble connecting to the validator list site. Check your firewall configuration to make sure you're not blocking outgoing traffic on port 80 (HTTP) or 443 (HTTPS). Also check that your DNS is able to resolve the domain of your validator list site.
+
+<!-- TODO: create a tutorial for how to sideload a validator list from file and link it here -->
+
+
+## Not Enough Peers
+
+If your server does not connect to enough [peer servers](peer-protocol.html), it may not be able to download enough data to remain synced with the network as the network continues processing new transactions. This can happen if your network connection is unreliable, or if you configure your server as a [private server](peer-protocol.html#private-peers) without adding enough reliable fixed peers.
+
+Use the [peers method][] to get information about your server's current peers. If you have exactly 10 or 11 peers, that may indicate that your firewall is blocking incoming peer connections. [Set up port forwarding](forward-ports-for-peering.html) to allow more incoming connections. If your server is configured as a private server, double-check the contents and syntax of the `[ips_fixed]` stanza in your config file, and add more proxies or public hubs if possible.
+
+
+## Corrupt Databases
+
+In rare cases, corrupt data saved in your `rippled` server's internal databases could cause it to fail to sync. You can safely delete your server's databases in most circumstances as long as the server is not running. Corrupt data can be the result of a momentary hardware failure when copying or writing to disk, a more serious disk failure, a different process crashing and writing to the wrong part of the disk, or other issues.
+
+For a more temporary test, you can temporarily change the paths to your server's databases as long as you have enough free space to re-download the current ledger and store other settings.
```

Comment:
I don't like the duplication of "temporary". How about "As a test, you can temporarily change the paths...."